userguide/system.rst

```diff
@@ -742,87 +743,153 @@ Three types of SED devices are supported:
+physically maintaining separate passwords for each SED.
+
+
+**Setting a global password for SEDs**
+
+Go to
+:menuselection:`System --> Advanced --> SED Password`
+and enter the password. **Record this password and store it in a safe
+place!**
+
+Now the SEDs must be configured with this password. Go to the
+:menuselection:`Shell`
+and enter :samp:`sedhelper setup {password}`, where *password* is the
+global password entered in
+:menuselection:`System --> Advanced --> SED Password`.
 
-Remember to take ownership of the devices:
+:command:`sedhelper` ensures that all detected SEDs are properly
+configured to use the provided password. Example:
 
 .. code-block:: none
 
- sedhelper setup password
+   root@truenas1:~ # sedhelper setup abcd1234
+   da9			[OK]
+   da10			[OK]
+   da11			[OK]
+
+
+Rerun :samp:`sedhelper setup {password}` every time a new SED is placed
+in the system to apply the global password to the new SED.
+
+
+**Creating separate passwords for each SED**
+
+Go to
+:menuselection:`Storage --> Disks`.
+Click |ui-options| for the confirmed SED, then :guilabel:`Edit`.
+Enter and confirm the password in the :guilabel:`SED Password` and
+:guilabel:`Confirm SED Password` fields. Disks that have a configured
+SED password show bullets in their row of the :guilabel:`SED Password`
+column of
+:menuselection:`Storage --> Disks`.
+Conversely, the rows in that column will be empty for disks that do not
+support SED or are unlocked using the global password.
+
+Now configure the SED to use the password. Go to the
+:menuselection:`Shell`
+and enter :samp:`sedhelper setup --disk {da1} {password}`, where *da1*
+is the SED to configure and *password* is the created password from
+:menuselection:`Storage --> Disks --> Edit Disks --> SED Password`.
+
+**Repeat this process for each detected SED and any SED deployed to the
+%brand% system in the future.**
+
+.. danger:: It is important to remember SED passwords! Without them,
+   SEDs cannot be unlocked and their data is unavailable. While it is
+   possible to specify the PSID number on the label of the device with
+   :command:`sedutil-cli`, doing so **erases the contents** of the
+   device rather than unlock it. Always record SED passwords whenever
+   they are configured or modified and store them in a safe place!
+
+
+.. _Check SED Functionality:
 
-This command ensures that all detected SED disks are properly setup using
-the specified password.
+Check SED Functionality
+^^^^^^^^^^^^^^^^^^^^^^^
 
-.. note:: Rerun :command:`sedhelper setup password` every time a new SED
-   disk is placed in the system.
+When SED devices are detected during system boot, the middleware
+checks for configured global and device-specific passwords. Devices with
+individual passwords are unlocked with their password and any remaining
+devices that have no device-specific password are unlocked using the
+global password. No additional manual intervention is required, but
```

Comment:
The second and following sentences should be split into a separate paragraph about using a mixed pool.